### PR TITLE
Fix bug

### PR DESCRIPTION
**Pull request by shuaiLiWang**
## What does this implement/fix? Explain your changes.
commit1：优化删除钱包慢的问题
commit2：恢复钱包后立刻删除所有钱包会报错，原因是recovery_comfirmed的时候对恢复的钱包进行了stop操作
commit3：https://github.com/OneKeyHQ/TaskHub/issues/1325

## Does this close any currently open issues?  
If it fixes a bug or resolves a feature request, be sure to link to that issue.
https://github.com/OneKeyHQ/TaskHub/issues/1325

## Pull request type
 
_Put an `x` in the boxes that apply_
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## Where has this been tested?
Python and ios

## Any other comments?
…

### CODE REVIEW

**Review by taimanhui — COMMENTED** [review]
需要更新

File: electrum_gui/android/console.py [hunk 1]
```diff
@@ -4241,8 +4241,15 @@ def delete_wallet_from_deamon(self, name):
...
-            return bool(txids)
+            with self.pywalib.override_server(self.coins[coin]):
+                try:
+                    address_info = self.pywalib.get_address(wallet_obj.get_addresses()[0])
+                except Exception:
+                    address_info = None
+            if address_info is not None and address_info.existing:
+                return True
+            else:
+                return False
```
[the remark this answers is not in the document]
Comment (shuaiLiWang) [reply electrum_gui/android/console.py]: 已更新 
这个函数是内部使用 所以 名称拼写错误已经修复

File: electrum_gui/android/console.py [hunk 2]
```diff
@@ -3182,8 +3182,8 @@ def recovery_confirmed(self, name_list, hw=False):
         name_list = json.loads(name_list)
         if len(name_list) != 0:
             for name in name_list:
-                if self.recovery_wallets.__contains__(name):
-                    recovery_info = self.recovery_wallets.get(name)
+                if self.recovery_wallets.get(name):
```
[the remark this answers is not in the document]
Comment (shuaiLiWang) [reply electrum_gui/android/console.py]: pop这行昨天我试过 但是 运行过程出会出错（会生成key.tmp临时钱包文件） 所以 我给放到下边去了 由于今天发版还有其他bug 这里我得有时间了查出错的具体原因

**Comment by shuaiLiWang** [comment]
这个加快哦 今天要打包~

**Review by huazhouwang — APPROVED** [review]

File: electrum_gui/android/console.py [hunk 3]
```diff
@@ -4225,12 +4225,19 @@ def reset_wallet_info(self):
...
-    def delete_wallet_devired_info(self, wallet_obj, hw=False):
+    def delete_wallet_derived_info(self, wallet_obj, hw=False):
         coin = wallet_obj.coin
         if coin in self.coins:
             # TODO: try implementing get_history for eth wallets.
-            chain_code = self._coin_to_chain_code(coin)
-            history = provider_manager.search_txids_by_address(chain_code, wallet_obj.get_addresses()[0])
+            with self.pywalib.override_server(self.coins[coin]):
+                try:
+                    address_info = self.pywalib.get_address(wallet_obj.get_addresses()[0])
```
Comment (taimanhui) [inline electrum_gui/android/console.py:4236]: pywalib的get_address已经删了，直接用`provider_manager.get_address`，也不需要override_server其它逻辑一样

**Review by taimanhui — APPROVED** [review]

**Pull request merged**